### PR TITLE
Export Academy app

### PR DESCRIPTION
**Pull request by bsabhilash**
This changeset includes a bare-bones, first iteration of the Export Academy app.

The app is behind a feature flag. To enable it, the `FEATURE_EXPORT_ACADEMY` env var needs to be set to true in the application's environment like this `FEATURE_EXPORT_ACADEMY=true`.

image
image
image
image
image

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-268
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s)

### Housekeeping

- [X] Added all new environment variables to Vault.

### Merging

- [X] This PR can be merged by reviewers.

### CODE REVIEW

File: export_academy/forms.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    job_title = forms.CharField(
+        label=_('Job title'),
+        max_length=50,
+        error_messages={
+            'required': _('Enter your job title'),
+        },
+    )
+    business_name = forms.CharField(
+        label=_('Business name'),
+        max_length=50,
```
Comment (mforner13) [inline export_academy/forms.py:39]: I think 50 could be too little for a business name - is there a reason to limit it this low?

File: export_academy/forms.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        choices=COUNTRIES,
+    )
+    like_to_discuss = forms.ChoiceField(
+        label=_('Radio button example'),
+        choices=(
+            ('no', 'No'),
+            ('yes', 'Yes'),
+        ),
+        widget=forms.RadioSelect,
+        error_messages={'required': _('Please answer this question')},
```
Comment (mforner13) [inline export_academy/forms.py:69]: This error message needs to follow the standard gov uk design - here's the guidance for radios - https://design-system.service.gov.uk/components/radios/#error-messages.

Might be best to double check with Neil :)

File: export_academy/models.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        MultiFieldPanel(
+            heading='Details',
+            children=[
+                FieldPanel('name'),
+                FieldPanel('description'),
+                FieldPanel('link'),
+            ],
+        ),
+        MultiFieldPanel(
+            heading='Date',
```
Comment (mforner13) [inline export_academy/models.py:40]: Could `'Timing'` or `'Dates'` be a better heading here as there are multiple dates?

File: export_academy/models.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    CONFIRMED = 'Confirmed'
+    CANCELLED = 'Cancelled'
+    STATUSES = (
+        (CONFIRMED, CONFIRMED),
+        (CANCELLED, CANCELLED),
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    event = ParentalKey(Event, on_delete=models.CASCADE, related_name='bookings')
+    registration = models.ForeignKey(Registration, on_delete=models.CASCADE)
```
Comment (mforner13) [inline export_academy/models.py:87]: Does this mean the person who has signed up for the event? If so I think `attendee` could be a clearer field name

File: export_academy/models.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    )
+
+    class Meta:
+        ordering = ('-created', '-modified')
+
+    def __str__(self):
+        return f'{self.id}:{self.name}'
+
+
+class Registration(TimeStampedModel):
```
Comment (mforner13) [inline export_academy/models.py:64]: I think this name implies it brings together one user registering for one event - which is what the Booking model does. What about `EventsUser`, `EventUser` or `Attendee` ?

File: export_academy/forms.py (hunk 1 above)
Comment (bsabhilash) [reply export_academy/forms.py:39]: We don't know yet. Copied from the existing company registration form. This is a draft, and we don't even know if this field is needed. I will clarify when the requirements are confirmed.

File: export_academy/forms.py (hunk 2 above)
Comment (bsabhilash) [reply export_academy/forms.py:69]: Thanks for the pointer. I'll make sure to clarify with Neil. We do not know whether a radio select is needed at this stage.

**Review by mforner13 — APPROVED** [review]

**Pull request merged**